production/ksonnet/loki/query-scheduler.libsonnet

```diff
@@ -0,0 +1,58 @@
+
+local k = import 'ksonnet-util/kausal.libsonnet';
+
+{
+  local container = k.core.v1.container,
+
+  // Override frontend and querier configuration
+  local override = {
+      frontend: {
+          scheduler_address: 'query-scheduler.%s.svc.cluster.local:9095' % $._config.namespace,
+      },
+      frontend_worker: {
+          frontend_address: '',
+          scheduler_address: 'query-scheduler.%s.svc.cluster.local:9095' % $._config.namespace,
+      },
+  },
+  _config +: {
```

Comment:
What's the best way to render this?

Reply:
What do you mean by `render` here?

Reply:
Generate the YAML and check that the override works.